api/src/routes/anime/index.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import animeRoutes from '@@root/routes/anime/animeRoutes';
+import { animeController } from "@@root/controllers";
+import { validation } from "@@middleware/index.js";
+
+const router = express.Router();
+
+router.use("/", animeRoutes);
+
+router.route("/id/:id")
```

Comment:
This can go back in the other file (`src/routes/anime/animeRoutes.ts`) and you can just add the `.post(...)` on the same route as `.get(..., controller.index)` since there is no `id` being used by the `create` method